### DISCUSSION TITLE
ACF Pro 6.2.5 - "shop_widget_content"

### DISCUSSION THREAD

**Discussion by StefanOtmar** [affiliatetheme/affiliatetheme#29] [General]
Seit dem Update auf ACF 6.2.5 erhalte ich im Wordpress Dashboard diese Meldung:
![Screenshot 2024-01-17 210511](https://github.com/affiliatetheme/affiliatetheme/assets/133593346/81752ae3-1c91-44d4-9562-f41fe47009c1)

Der Link "Learn how to fix" führt zu dieser Seite: https://www.advancedcustomfields.com/blog/acf-6-2-5-security-release/

Mir ist leider nicht klar was zu tun ist, bzw. ist das wahrscheinlich eher ein Thema, welches die Entwickler des Affiliate umsetzen müssen. Oder?

Viele Grüße

**Comment by endcoreCL** [comment]
Wir schauen uns das an und geben Feedback, ggf, kommt dazu ein kleiner Release.

**Comment by Chauffe** [comment]
Ist es denn generell sicher auf 6.2.5. zu updaten oder sollte man noch warten? Immerhin ist in der 6.2.4. ja eine Sicherheitslücke? Ich habe nur Angst, dass nach dem Update gar nichts mehr geht...

**Comment by endcoreCL** [comment]
Das ist nur ein Hinweis von ACF, das hat keine Auswirkungen auf die Funktion des Themes. Ein Update ist im Zusammenhang mit dem Theme kein Problem.

Die Ausgabe des Shop-Widgets muss angepasst werden, falls das ohnehin nicht genutzt wird, ist es sowieso nicht wichtig.

**Comment by StefanOtmar** [comment]
Ich kann bestätigen, dass das Theme auch mit ACF 6.2.5 ohne Probleme läuft.

Zusätzlich zum Shop-Widget betrifft es aber noch einen weiteren Bereich (siehe Screenshot):
![10](https://github.com/affiliatetheme/affiliatetheme/assets/133593346/6febf893-4e89-4fe6-9a25-a9f212d75cbb)

**Comment by endcoreCL** [comment]
Die Fehler sind mit dem kommenden Update behoben
